prow/integ-suite-kind.sh

```diff
@@ -126,6 +126,7 @@ export KIND_REGISTRY="localhost:${KIND_REGISTRY_PORT}"
 
 export HUB=${HUB:-"istio-testing"}
 export TAG="${TAG:-"istio-testing"}"
+export VARIANT
```

Comment:
Why are we exporting it here?

Reply:
Well it didn't work so apparently not for a good reason :-).

The intent is for our `make` call to have access to the `VARIANT` var set on the job, though